liwords-ui/src/gameroom/scorecard.tsx

```diff
@@ -280,59 +264,54 @@ export const ScoreCard = React.memo((props: Props) => {
     : undefined;
   const notepadStyle = cardHeight
     ? {
-        height: cardHeight - 24,
+        height: cardHeight - 60,
+        display: notepadHidden ? 'none' : 'block',
       }
     : undefined;
-  const title = notepadVisible ? 'Notepad' : `Turn ${turns.length + 1}`;
-  const extra = notepadVisible ? 'View Scorecard' : 'View Notepad';
+  let title = `Turn ${turns.length + 1}`;
+  let extra = null;
+  if (enableNotepadFlip) {
+    title = !notepadHidden ? 'Notepad' : `Turn ${turns.length + 1}`;
+    extra = !notepadHidden ? 'View Scorecard' : 'View Notepad';
+  }
   return (
     <Card
       className="score-card"
       title={title}
       // eslint-disable-next-line jsx-a11y/anchor-is-valid
       extra={
-        <button
-          className="link"
-          onClick={() => {
-            if (notepadVisible) {
-              setNotepadVisible(false);
-            } else {
-              setNotepadVisible(true);
-            }
-          }}
-        >
-          {extra}
-        </button>
+        isTablet ? (
+          <button
+            className="link"
+            onClick={() => {
+              if (notepadHidden) {
+                setNotepadHidden(false);
+              } else {
+                setNotepadHidden(true);
+              }
+            }}
```

Comment:
this can go to a `useCallback` and be succinctly described like so:

```jsx
const toggleNotepadVisibility = useCallback(() => {
  setNotepadHidden((x) => !x);
}, []);
```

stating the new state as a function of the old state makes the `[]` dependency list valid.

Reply:
Fixed